<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gallery</class>
 <widget class="QStackedWidget" name="gallery">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>980</width>
    <height>720</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>980</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>980</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <widget class="QWidget" name="page">
   <property name="minimumSize">
    <size>
     <width>980</width>
     <height>720</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>980</width>
     <height>720</height>
    </size>
   </property>
   <widget class="QPushButton" name="pushButton_16">
    <property name="geometry">
     <rect>
      <x>634</x>
      <y>510</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Siguiente Imagen</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIMGDESC">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>771</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_8">
    <property name="geometry">
     <rect>
      <x>154</x>
      <y>520</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Anterior Imagen</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelIMG">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>20</y>
      <width>781</width>
      <height>421</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>980</width>
      <height>540</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
